### ISSUE TITLE
Brownie becomes mandatory when importing eth_retry (ImportError)

### ISSUE THREAD

**Issue by manumonti** [BobTheBuidler/eth_retry#24]
For some reason I can't fully understand, when I install eth_retry with pip:

```bash
pip install eth_retry
```

This [confest.py](https://github.com/BobTheBuidler/eth_retry/blob/c253f1e8138ed6702c9952e7a7594e869117019d/tests/conftest.py) is installed in my virtual environment (pyenv) under the tests/ folder. Specifically in:

`~/.pyenv/versions/3.10.11/envs/myvirtualenv/lib/python3.10/site-packages/tests/conftest.py`

This `conftest.py` located on my virtualenv is executed when I use `pytest` in the package where I'm importing eth_retry. Such as I don't have brownie as dependency, `pytest` returns this error:

```log
ERROR: Failed setting default provider: Provider 'test' not found in network 'local'.
ImportError while loading conftest '/Users/manumonti/.pyenv/versions/3.10.11/envs/myvirtualenv/lib/python3.10/site-packages/tests/conftest.py'.
../../../../../nucypher/lib/python3.10/site-packages/tests/conftest.py:5: in <module>
    from brownie import network
E   ModuleNotFoundError: No module named 'brownie'
```

As far as I understand, Brownie is only needed for testing and not for the actual use of eth_retry.

Is there any way to avoid this error when using pytest and importing eth_retry?

This behavior has been found with eth_retry v0.1.17 and v0.1.18.

**Comment by manumonti** [comment]
Maybe a simple solution could be not adding the `tests/` folder to the package uploaded to Pypi.

This can be made by using package exceptions in `setup.py`:

https://setuptools.pypa.io/en/latest/userguide/quickstart.html#package-discovery

**Comment by BobTheBuidler** [comment]
Testing a fix for this on #30

**Comment by BobTheBuidler** [comment]
fixed in [0.1.20](https://github.com/BobTheBuidler/eth_retry/releases/tag/v0.1.20)
